package.json

```diff
@@ -30,51 +50,26 @@
     "redux": "^4.0.0",
     "redux-promise-middleware": "^5.0.0",
     "redux-thunk": "^2.2.0",
-    "sanitize-html": "1.x",
-    "showdown": "1.8.x",
-    "version-sort": "0.1.x"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.1.6",
-    "@babel/plugin-proposal-class-properties": "^7.1.0",
-    "@babel/preset-env": "^7.1.6",
-    "@babel/preset-react": "^7.0.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^8.2.6",
-    "babel-loader": "^8.0.4",
-    "bootstrap-sass": "^3.3.7",
-    "cache-loader": "1.x",
-    "css-loader": "^1.0.1",
-    "eslint": "^4.19.1",
-    "eslint-plugin-patternfly-react": "^0.2.0",
-    "file-loader": "^2.0.0",
-    "font-awesome": "4.7.x",
-    "html-webpack-plugin": "^3.2.0",
-    "mini-css-extract-plugin": "^0.4.5",
-    "node-sass": "4.8.x",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "prettier": "^1.14.2",
     "regenerator-runtime": "^0.13.1",
     "resolve-url-loader": "2.x",
     "rimraf": "^2.6.2",
-    "style-loader": "^0.23.1",
+    "sanitize-html": "1.x",
     "sass-loader": "6.x",
+    "showdown": "1.8.x",
+    "style-loader": "^0.23.1",
     "thread-loader": "1.x",
     "uglifyjs-webpack-plugin": "^2.0.1",
-    "webpack": "^4.25.1",
+    "version-sort": "0.1.x",
+    "webpack": "^4.29.0",
     "webpack-bundle-analyzer": "^3.0.3",
     "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.10",
+    "webpack-dev-server": "^3.1.14",
     "webpack-merge": "^4.1.4"
   },
   "scripts": {
-    "prebuild": "yarn clean",
-    "build": "webpack --config webpack.prod.js",
-    "start": "webpack-dev-server --hot --color --progress --info=true --config webpack.dev.js",
-    "mock": "webpack-dev-server --hot --color --progress --info=true --config webpack.dev.js --define process.env.MOCK_MODE='\"true\"'",
```

Comment:
We currently need mock mode to have a work around available for the github api throttle